Library/Homebrew/cmd/link.rb

```diff
@@ -24,15 +24,17 @@ def link
     mode.dry_run = true if ARGV.dry_run?
 
     ARGV.kegs.each do |keg|
-      if HOMEBREW_PREFIX.to_s == "/usr/local" && keg.name == "openssl"
-        opoo <<-EOS.undent
-          Refusing to link: openssl
-          Linking keg-only OpenSSL means you may end up linking against the insecure,
-          deprecated system version while using the headers from the Homebrew version.
-          Instead, pass the full include/library paths to your compiler e.g.:
-            -I#{HOMEBREW_PREFIX}/opt/openssl/include -L#{HOMEBREW_PREFIX}/opt/openssl/lib
-        EOS
-        next
+      if HOMEBREW_PREFIX.to_s == "/usr/local"
+        if keg.name.start_with?("openssl") || keg.name.start_with?("libressl")
```

Comment:
Part of me wants to get Travis building in `/opt/brew` as well so we know how much breaks in that directory. I'd generally like us to stop vilifying non-`/usr/local` installations & think it'd be reasonably easy to support, as well as opening up the option of moving in future if Apple clamps down even more or we decide it's worthwhile.

But that's not strictly related to this discussion, so I'll leave that be.